install_config/persistent_storage/selector_label_binding.adoc

```diff
@@ -111,6 +111,58 @@ spec:
 labels of targeted PV.
 ====
 
+[[selector-label-volume-bind]]
+=== Optional: Bind the PVC to a specific PV when using a selector
+
+As the *cluster-admin* user, you can bind the PVC to a specific PV only.
+
+To reserve a PV for specific tasks, you have two options: create a specific storage class, or pre-bind the PV to your PVC.
+
+. Request a specific storage class for the PV by specifying the storage class’s name.
++
+The following resource shows the required values that you use to configure a StorageClass. This example uses the AWS ElasticBlockStore (EBS) object definition.
+
++
+.StorageClass definition for EBS
+====
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: gp2
+provisioner: kubernetes.io/aws-ebs
+...
+----
+====
++
+[IMPORTANT]
+====
+If necessary in a multi-tenant environment, use a quota definition to reserve the storage class and PV(s) only to a specific namespace.
```

Comment:
Should this be called out like this and will this statement be clear to a cloud admin?